scripts/nikon_shift_correction.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
```

Comment:
It's not clear what this script is for. It feels like it's a mix between analyse_shifts.py and curvefit_test.py.